lib/dbs/firebase.ts

```diff
@@ -53,10 +58,15 @@ export async function setStore(session: SessionProps) {
 // User management for multi-user apps
 // Use setStoreUser for storing store specific variables
 export async function setStoreUser(session: SessionProps) {
-    const { access_token: accessToken, context, owner, user: { id: userId } } = session;
+    const {
+        access_token: accessToken,
+        owner,
+        user: { id: userId },
+    } = session;
     if (!userId) return null;
 
-    const storeHash = context?.split('/')[1] || '';
+    const contextString = session?.context ?? session?.sub;
```

Comment:
since we're already destructuring `session`, let's add `context` and `sub`
```suggestion
    const contextString = context ?? sub;
```